app/labor/code_block_parser.rb

```diff
@@ -93,7 +93,7 @@ def escape_colon_emojis_in_codeblock(html)
       next if el.name == "code"
 
       if el.search("code").empty?
-        el.swap(Html::ParseEmoji.call(el.to_html))
+        el.swap(Html::ParseEmoji.call(el.to_html)) if el.parent.present?
```

Comment:
There is already a spec written for this feature which still passes for this change. From what I understand, this code was trying to perform swap on parentless text dom that mainly consists of newline ("\n\n\n\n\n"). It was never meant to work so they added a proper `RuntimeError`